documents/src/pages/utils/cache.md

```diff
@@ -0,0 +1,133 @@
+<!-- 
+title: Local caching Utility
+location: ./utils/cache
+type: page
+layout: default
+-->
+
+::status-complete::
+
+# Local cache
+The utility provides provides an alternative way to store data on your local machine.
+There are 2 main classes that collaborate: wrapper and storage. LocalCache is high level class that wraps the selected storage to be asynchronous api.
+The second is storage class. Store data on the user's machine. The storages has a list of support as below:
+- localstorage
+- indexeddb
+
+LocalCache can select only one storage to use. Here is a common use.
+
+```typescript
+import { LocalCache } from '@refinitiv-ui/utils/cache.js';
+// Create cache and use the storage
+const cache = new LocalCache(
+  // prefix key for localstorage or database name for indexeddb
+  'my-cache',
+  { storage: 'indexeddb' } // config with storage type
+);
+// Example using an api
+await cache.set(key, data);
+const item = await cache.get(key);
+```
+
+## LocalCache APIs
+
+### set
+Caches a value against a key to use until expired
+
+```typescript
+import { LocalCache } from '@refinitiv-ui/utils/cache.js';
+const key = 'my-item-01';
+const value = { label: '01', value: 1};
+// Create cache and use the storage
+const cache = new LocalCache('my-cache', { storage: 'indexeddb' });
+// store 'my-item-01': { label: '01', value: 1} to LocalCache. Default cache expiry is 5 days
+await cache.set(key, value);
+```
+
+#### Syntax
+
+```text
+cache.set(key, value, expires);
+```
+
+#### Arguments
+
+| Name | Type | Description |
+| --- | --- | --- |
+| key | String | Cache key |
+| value | unknown | Data to store in cache |
+| expires | Number | Cache expiry in seconds. Defaults to 5 days. |
```

Comment:
I think there are less verbose ways to document cache APIs. I personally find something like TS declaration much easier to understand. 

```ts
export declare class LocalCache {
    /**
     * Storage to store data
     */
    protected storage: CacheStorage;
    constructor(name: string, config: LocalCacheConfig);
    /**
     * Caches a value against a key to use until expired
     * @param key Cache key
     * @param value Data to store in cache
     * @param [expires=432000] Cache expiry in seconds. Defaults to 5 days.
     * @returns Promise void
     */
    set(key: string, value: string, expires?: number): Promise<void>;
    /**
     * Returns cache data value based on provided key
     * @param key Cache key
     * @returns Promise string data or `null` if nothing is cached
     */
    get(key: string): Promise<unknown | null>;
    /**
     * Remove cache data value based on provided key
     * @param key Cache key
     * @returns Promise void
     */
    remove(key: string): Promise<void>;
    /**
     * Clear all memory cache
     * @returns Promise void
     */
    clear(): Promise<void>;
}
```